packages/repository-tests/src/crud/relations/fixtures/models/order.model.ts

```diff
@@ -24,14 +26,15 @@ export class Order extends Entity {
   @property({
     type: 'boolean',
     required: false,
+    default: false,
```

Comment:
Can we remove this default value please? I think it should not be needed after the changes you made recently, based on my older comments.